src/simulator/src/arrayHelpers.ts

```diff
@@ -4,7 +4,7 @@
 export default Array = window.Array
 
 Object.defineProperty(Array.prototype, 'clean', {
-    value: function (deleteValue) {
+    value: function (deleteValue: any) {
```

Comment:
We should avoid using `any`, use the correct type or if it is unknown to you use `unknown` instead. But would encourage to use the correct type.